README.md

```diff
@@ -0,0 +1,50 @@
+# crispr_tools
+Collection of tools for CRISPR-Cas9 genetic screen analysis.
+
+## Introduction
+These scripts neatly pipeline read counting from FASTQ and wrap differential analysis with DrugZ and MAGeCK in batch.
+
+## Pipeline overview
+CRISPR screen analysis using crispr_tools proceeds by the following steps:
+1. From pre-aligned FASTQ, determine read counts per protospacer and map them to the reference CRISPR library.
+2. Specify replicate-to-condition mappings in the experimental design and select differential methods to run.
+3. Run the analysis in batch.
+
+# Tutorial
+A tutorial dataset is provided in the `tut` directory. To verify that you have correctly run the analysis, your directory should resemble `tut_ran`. Follow the steps below for a detailed explanation of the analysis.
+
+## Reading counts
+The `count_reads.py` script handles read counting from pre-aligned FASTQ files and mapping to a reference library, if supplied.
+
+`count_reads.py [FASTQ_FILES] -s SLICE --merge-samples --just-go -p OUTDIR --library=LIB`
+
+The slice (`SLICE`) is range that specifies the position of the protospacer along the reads. It is supplied as zero-based half-open indices. FASTQ files must be pre-aligned and have a constant slice across all reads in all runs.
+
+The slice can be determined by inspecting nucleotide frequencies as a function of cycle. It appears as a region of uniform probability across the four bases. The `ntByCycle.R` script in the exorcise package (https://github.com/SimonLammmm/exorcise/) can generate such nucleotide traces.
+
+`ntByCycle.R -f [FASTQ_FILE] -o OUTDIR`
+
+Specify `--merge-samples` if you have technical replicates. You'll know this is the case if you have FASTQ files with `_L001` and `_L002` or similar.
```

Comment:
I think you're technically right that these are a kind of technical replicate, but it might confuse people.  On the sequencer in the gurdon they were lanes in the sequencer that recieve from the same physical sample... This option might be better named --merge-lanes, but maybe it's a bit late for that.